public/images/ub04.svg

```diff
@@ -3,7 +3,7 @@
 "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
 <svg viewBox="0 0 934 1210" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
 <defs>
-<style type="text/css"><![CDATA[
+<style><![CDATA[
 .g1_1{
```

Comment:
Please confirm this file.

Reply:
This is an image. It shouldn't be messed with.

Reply:
OK, I added it.